File: bricks/pac_add_native_ui/__brick__/packages/{{name.snakeCase()}}/android/src/main/kotlin/com/danhdue/{{name.snakeCase()}}/presentation/{{name.pascalCase()}}State.kt
/*
 * Copyright (c) 2026, one of DanhDue ExOICTIF projects. All rights reserved.
 */

/*
 * coverage:ignore-file
 */

package com.danhdue.{{name.snakeCase()}}.presentation

data class {{name.pascalCase()}}State(
    val isLoading: Boolean = false,
    val title: String = "{{name.pascalCase()}} Native UI",
    val error: String? = null
) : BaseState

File: bricks/pac_add_native_ui/__brick__/packages/{{name.snakeCase()}}/ios/Classes/Presentation/{{name.pascalCase()}}State.swift
public struct {{name.pascalCase()}}State: BaseState {
    public var isLoading: Bool = false
    public var title: String = "{{name.pascalCase()}} Native UI"
    public var error: String? = nil

    public init(isLoading: Bool = false, title: String = "{{name.pascalCase()}} Native UI", error: String? = nil) {
        self.isLoading = isLoading
        self.title = title
        self.error = error
    }
}
